package.json

```diff
@@ -111,68 +111,68 @@
     "npm-run-all": "^4.1.5",
     "postcss-loader": "^3.0.0",
     "rimraf": "^2.6.2",
-    "sass-loader": "^7.1.0",
+    "sass-loader": "^7.3.1",
     "script-loader": "^0.7.2",
     "style-loader": "^0.23.1",
-    "ts-jest": "^26.2.0",
+    "ts-jest": "^26.5.6",
     "ts-loader": "^5.3.1",
     "ts-node": "^7.0.1",
-    "tslint": "^5.12.0",
+    "tslint": "^5.20.1",
     "tslint-loader": "^3.6.0",
     "tslint-react": "^3.6.0",
```

Comment:
Given that `tslint` has been deprecated for some time now, you'll want to move off of these dependencies but you may want to wait for a separate PR for that as it requires additional configuration changes.